### DISCUSSION TITLE
Player Postion design discussion

### DISCUSSION THREAD

**Discussion by snow-QingYang** [CSC207-2023Y-UofT/course-project-monopoly#10] [Ideas]
I want to use Arraylist to store position data and create a Hashtable for each building.

**Comment by AlwynYin** [comment]
Structure of PositionImpactor
![IMG_D16FD9B876FE-1](https://github.com/CSC207-2023Y-UofT/course-project-monopoly/assets/73570489/3acbf988-1d50-429a-94a7-39d9042ad357)
